### PR TITLE
missing comment on exported function ConstructCustomDNSNames

### PR DESCRIPTION
**Pull request by elevran**
golint complains:
```sh
$ golint ./security/pkg/k8s/controller/customdnsname.go 
./security/pkg/k8s/controller/customdnsname.go:23:1: exported function ConstructCustomDNSNames should have comment or be unexported
```

### CODE REVIEW

File: security/pkg/k8s/controller/customdnsname.go [hunk 1]
```diff
@@ -20,6 +20,8 @@ import (
 	"istio.io/istio/pkg/log"
 )
 
+// ConstructCustomDNSNames creates DNS entries for given service accounts and allows
+// customization of the DNS names of specific service accounts
```
Comment (incfly) [inline security/pkg/k8s/controller/customdnsname.go]: would be helpful to indicate what and how customization will be generated

Comment (elevran) [reply security/pkg/k8s/controller/customdnsname.go]: agree. Added in additional commit.
@incfly PTAL

**Comment by esnible** [comment]
The required bookinfo test failed because Prometheus got stuck in `init:0/1`.  Can someone give me a positive review of https://github.com/istio/istio/pull/11904 so that the logs will tell the reason?  (Hint: it is always a lack of resources scheduling Prometheus pod).

**Pull request merged**